### PR TITLE
added a heelo mesage and a gif of a typing cat

### PR DESCRIPTION
**Pull request by kt-lmb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -492,3 +492,10 @@ I think it should be here.
 *first-time on the kottans pre-course:*
 
 ![first-time on the kottans pre-course](gif/first-time-kottans.gif)
+
+
+
+
```
Comment (yevhenorlov) [inline README.md]: you don't need all this empty space, markdown removes in automatically on display
one empty line is sufficient :slightly_smiling_face:

**Comment by OleksiyRudenko** [comment]
Meanwhile the repo's been updated. Please, resolve the conflict.
If you need any help, please, let us know.

**Comment by OleksiyRudenko** [comment]
@kt-lmb , you are almost done!
Remove images from your local repo (hint: use `git rm <filename> <path/filename>`)
Then commit and push.
A tiny step and you're on the top!

P.S. It might be challenging but the benefit for you is that level-up your git skills!
I am eager to merge your PR 😉

**Comment by OleksiyRudenko** [comment]
@kt-lmb , you have successfully removed `./gif/typingCat.gif` and also tried to remove `./gif/giphy.gif`. But the latter has no relative path. Just do `git rm giphy.gif` as it is located in the root of your project.

Now there is also a formal merge conflict as `README.md` has been changed.

Will you resolve it? You need to keep only one of two alternatives (hint: the second one is empty but it is in place. Sounds weird, I know). Built-in github conflict resolver is very helpful.

**Comment by OleksiyRudenko** [comment]
@kt-lmb grats! You did it!

**Pull request merged**